rs/bitcoin/ckbtc/minter/src/memo.rs

```diff
@@ -14,7 +14,7 @@ pub enum Status {
     #[n(0)]
     /// The minter accepted a retrieve_btc request.
     Accepted,
-    /// The minter rejected a retrieve_btc due to a failed KYT check.
+    /// The minter rejected a retrieve_btc due to a failed bitcoin check.
```

Comment:
```suggestion
    /// The minter rejected a [retrieve_btc] request due to a failed Bitcoin check.
```

Reply:
Yes, let me quickly do it. Thanks for pointing out the difference